src/pages/feed/RecentsFeed/RecentsFeed.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { push } from 'react-router-redux'
+import { connect } from 'react-redux'
+import Icon from '@santiment-network/ui/Icon'
+import { ProjectSelector } from '../../../ducks/SANCharts/Header'
+import { SidecarItems } from '../../../ducks/SANCharts/ChartSidecar'
+import styles from './RecentsFeed.module.scss'
+
+const RecentsFeed = ({ onSlugSelect }) => {
+  return (
+    <div className={styles.container}>
+      <div>
+        <ProjectSelector
+          trigger={() => (
+            <div className={styles.search}>
+              Explore assets
+              <Icon type='search' className={styles.searchIcon} />
+            </div>
+          )}
+        />
+      </div>
+      <div className={styles.scrollable}>
+        <SidecarItems
+          onSlugSelect={onSlugSelect}
+          onProjectClick={onSlugSelect}
+          classes={styles}
+          params={{
+            gainersTitle: 'Gainers and loosers'
```

Comment:
Let's use one heading for that layout on all pages. We used 'Social gainers and losers` in other places
@Partysun ?

Reply:
@GarageInc Alan approved using the same naming: `Social gainers and losers`